interface/patient_file/summary/demographics_full.php

```diff
@@ -744,6 +737,12 @@ function end_group() {
 <!-- include support for the list-add selectbox feature -->
 <?php include $GLOBALS['fileroot']."/library/options_listadd.inc"; ?>
 
+<?php /*Include the validation script and rules for this form*/
+$form_id="DEM";
```

Comment:
I guess this will be more clear when I test. Do you have any constraints defined for the DEM layout?